tests/testfiles/CommonMark_0.30/links/051.test

```diff
@@ -15,7 +15,6 @@ BEGIN link
 - URI: /uri
 - title: 
 END link
-interblockSeparator
 BEGIN link
 - label: ref
 - URI: /uri
```

Comment:
`[bar]` is a collapsed reference link, whereas `[ref]: /uri` should produce no output.
The HTML output in the unit test is correct in this respect.